packages/app/src/cli/models/app/loader.ts

```diff
@@ -78,6 +76,11 @@ export async function loadConfigurationFile(
   }
 }
 
+const isVersionedAppConfigurationSchema = (content: {[key: string]: unknown}) => {
+  if (content.version && content.version === '3') return true
+  return false
```

Comment:
```suggestion
  return content.version === '3'
```

What is this version and where is it coming from?

Reply:
This is a temp solution, it's representing the `version` of the config schema. The CLI adds it automatically for the apps that use `app_config` modules. I thought it's a better way of checking the schema used instead of parsing the content against the different schemas. However it's subject to changes

Reply:
Finally this `version` field is gone as we are going to support the current app config toml schema